src/impl/peerconnection.cpp

```diff
@@ -1301,6 +1313,13 @@ void PeerConnection::resetCallbacks() {
 	trackCallback = nullptr;
 }
 
+std::vector<struct RemoteFingerprint> PeerConnection::remoteFingerprints() {
+	std::vector<struct RemoteFingerprint> ret;
+	ret = rFingerprints;
```

Comment:
I'm trying to return a copy of the vector here, I'm not sure if this is sufficient or even necessary?